shell/initialize/app-extended.js

```diff
@@ -73,41 +47,41 @@ async function createApp() {
         return err;
       }
     },
-    ...App
+    ...AppView
```

Comment:
This is the standard way to call Views in Vue to avoid confusion, plus it clashed with other vars.